skills/hears.js

```diff
@@ -1,39 +1,59 @@
 const { Skill } = require('../models/schema');
 const { UniqueViolationError } = require('objection-db-errors');
 
+
+const extractSkills = (messageString) => {
+  var skillPattern = /_[^@_]+_/g;
+  var matches = messageString.match(skillPattern);
```

Comment:
Nice Pattern!

Reply:
But I would suggest to use `const` instead of `var`, cause that would potentially avoid some issues, [this article](https://hackernoon.com/bye-bye-var-let-and-const-in-es6-afd91704f0ee) has some good examples.

Reply:
@ZhangChen199102 yep, I really should read more about var/const/let I don't know the difference, good pick up.